src/fparser.ts

```diff
@@ -842,6 +911,25 @@ export default class Formula {
                 idx++;
             }
         }
+
+        // Replace all Logical expressions with a partial tree:
+        idx = 0;
+        expr = null;
+        while (idx < exprCopy.length) {
+            expr = exprCopy[idx];
+            if (expr instanceof LogicalExpression) {
+                if (idx === 0 || idx === exprCopy.length - 1) {
+                    throw new Error('Wrong operator position!');
+                }
+                expr.left = exprCopy[idx - 1];
+                expr.right = exprCopy[idx + 1];
+                exprCopy[idx - 1] = expr;
+                exprCopy.splice(idx, 2);
+            } else {
+                idx++;
+            }
+        }
+
```

Comment:
See comment above: Please change the order of logical operator precedence to fit between PowerExpression and MultDivExpression:

I think the "correct" or logical precedence of operators is:

    (strongest): Power of (x^y)
    logical operators (x<y)
    multiplication/division
    additon/ substraction

So the following formula:

3+2>1*4<2^3

should be parsed as

3+((2>1)*(4<(2^3)))

Reply:
On the other hand, if we look at other programming languages, e.g. JavaScript, the logical operators have the lowest precedence:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence#table

What do you think, what would be an appropriate handling / precedence? I would recommend to implement the general consensus, as in other languages...